examples/mdetr/phrase_grounding.json

```diff
@@ -0,0 +1,8 @@
+{
+    "combine_datasets": ["flickr"],
+    "combine_datasets_val": ["flickr"],
+    "GT_type" : "separate",
+    "flickr_img_path" : "",
+   "flickr_dataset_path" : "",
+   "flickr_ann_path" : ""
```

Comment:
are these supposed to be empty? I see `flickr_dataset_path` is used but this is not defined

Reply:
Yeah I am following how they do things in the MDETR repo here. The user needs to fill this in with the local path containing the relevant files. Leaving it empty as I don't want to use my own local filepaths